packages/site/src/utils/guides.js

```diff
@@ -0,0 +1,42 @@
+export default {
```

Comment:
Is this file expected to live here long term or is it for mock-up purposes only?  If it is long term, than it is probably not a "util".  Also, consider having a separate title or name field from the keys and avoid having spaces in the keys.